examples/decomposition/topics_extraction_with_online_lda.py

```diff
@@ -0,0 +1,71 @@
+"""
+==================================================
+Topics extraction with Latent Dirichlet Allocation
+==================================================
+
+This is an example of showing how to extrat topics from
+20 newsgroup dataset with OnlineLDA. Also, It shows how 
+top words in each topic changed during training.
+
+The code is modified from
+"Topics extraction with Non-Negative Matrix Factorization"
+example.
+
+"""
+
+# Authors: Chyi-Kwei Yau
+
+
+from sklearn.datasets import fetch_20newsgroups
+from sklearn.feature_extraction.text import CountVectorizer
+from sklearn.utils import gen_batches
+from sklearn.decomposition import LatentDirichletAllocation
+
+
+def online_lda_example():
+    """
+    Example for LDA online update
+    """
+
+    # In default, we set topic number to 10, and both hyperparameter
```

Comment:
"By default, we set the number of topics to 10". If it is by default, why do we set it here?